CHANGELOG

```diff
@@ -19,6 +19,14 @@
   they stand out in the output.
   Thanks Gabriel Reis for the PR.
 
+- add more talks to the documentation
+- extend documentation on the --ignore cli option 
+- use pytest-runner for setuptools integration 
+- minor fixes for interaction with elcapitan system integrity protection (thanks Florian)
```

Comment:
Might want to say "OS X El Capitan" here so it's clear what this means, even I had to double-read it to understand :laughing: